.github/workflows/linux.yml

```diff
@@ -497,3 +497,44 @@ jobs:
         PKG_CONFIG_PATH="$PWD" pip install . -vv --no-build-isolation
         pushd $RUNNER_TEMP
         PYTHON_GIL=0 python -m pytest --pyargs scipy -n2 --durations=10
+  #################################################################################
+  clang_17_build:
+    # Purpose is to check for warnings in builds with latest clang.
+    # We do not run the test suite here.
+    name: Clang-17 build, fast, py3.12
```

Comment:
I think 'fast' is supposed to refer to a fast (rather than full) test suite run, so should probably be omitted here.

Reply:
Good catch. Yes, definitely. Just something that got left in when I copied this over.